packages/flutter/lib/src/material/text_field.dart

```diff
@@ -113,6 +129,18 @@ class _TextFieldSelectionGestureDetectorBuilder extends TextSelectionGestureDete
           Feedback.forLongPress(_state.context);
           break;
       }
+
+      switch (targetPlatform) {
+        case TargetPlatform.android:
+        case TargetPlatform.iOS:
+          editableText.showMagnifier(details.globalPosition);
+          break;
+        case TargetPlatform.fuchsia:
+        case TargetPlatform.linux:
+        case TargetPlatform.macOS:
+        case TargetPlatform.windows:
+          break;
+      }
```

Comment:
Does any of this need to be done for CupertinoTextField somewhere too?

Reply:
Looks like no, because it's also included in the parent class that CupertinoTextField is using directly 👍

Reply:
> 

Not sure why its like that, but it definitely doesn't have to be.